roles/test-install/tasks/main.yml

```diff
@@ -2,31 +2,43 @@
 
 - name: 'Remove {{ beat_pkg_name }} rpm if it exists'
   yum: name={{ beat_pkg_name }} state=absent lock_timeout=900
+  retries: 3
+  delay: 10
```

Comment:
This should mitigate the issues when the third party systems are a bit flaky, I've got a couple of issues in a couple of builds so this should give a bit more resilience.